tasks/managed/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -67,7 +67,7 @@ spec:
         set -eo pipefail
 
         SNAPSHOT_PATH=$(workspaces.data.path)/$(params.snapshotPath)
-        RESULTS_FILE="$(workspaces.data.path)/$(params.resultsDirPath)/add-fbc-contribution-results.json"
+        RESULTS_FILE="$(workspaces.data.path)/$(params.resultsDirPath)/add-fbc-contribution-$(params.pipelineRunUid)-results.json"
```

Comment:
Why do you think this is needed? We still use the subdirectory for everything I think. But more importantly, we have a separate PVC for each PLR. And other tasks don't do anything special either, at least push-snapshot just has `RESULTS_FILE="$(params.dataDir)/$(params.resultsDirPath)/push-snapshot-results.json"`

Reply:
Maybe I looked to the wrong task to see if this was still required or not. I will crosscheck the other tasks again, but you are probably right that this is not required.

Reply:
removed.